packages/www/components/GettingStarted/index.tsx

```diff
@@ -140,7 +141,7 @@ curl --request POST \
           </Heading>
         </Flex>
         <Link href={appendProjectId("/settings")} passHref legacyBehavior>
-          <Button as="a" size="2">
+          <Button className="bg-accent" as="a" size="2">
```

Comment:
Same thing here, we should have a handful of consistent Button styles, and then use that to re-style the dash.

Reply:
Yep, with `shadcn` we can do that, but as we are still didn't migrated all component, we can keep them as it is and just change color